README.rst

```diff
@@ -81,22 +81,19 @@ screenshot
    -s do not capture output (--capture=no)
    -t do not cut tracebacks (--full-trace)
    -v verbose (--verbose)
+   -w very verbose (--verbose --verbose)
    -x exit after first failure (--exitfirst)
 
   Options
-   =k only names matching expression (-k)
-   =m only marks matching expression (-m)
+   =k only names matching expression (-k=)
+   =m only marks matching expression (-m=)
    =t traceback style (--tb=)
-   =x exit after N failures or errors (--maxfail="10")
+   =x exit after N failures or errors (--maxfail=)
 
-  Run tests
-   t Test all                r Repeat last test run    x Test last-failed
-
-  Run tests for specific files
-   f Test file (dwim)       F Test this file         m Test multiple files
-
-  Run tests for current function/class
-   d Test def/class (dwim)    D Test this def/class
+  Run tests                 Run tests for specific files              Run tests for current function/class
```

Comment:
i guess some of these headings and command descriptions can be shortened, but that can be done later as well

Reply:
Happy to update, if you have suggestions.

I forgot to mention that `Switches` and `Options` are no longer mandatory, nor required by `transient` to achieve different behaviour. And that can be reorganised as well.

Reply:
thanks, good ideas.

 let's play with that later. it's unrelated to the library dependency switch itself.

Reply:
fwiw, see #28 for follow-up